packages/docs-utilities/index.d.ts

```diff
@@ -4,4 +4,4 @@ export function fixBabelGeneratorIssues(source: string): string;
 
 export function fixLineEndings(source: string, target: string): string;
 
-export function getUnstyledFilename(filename: string, definitionFile: boolean = false): string;
+export function getUnstyledFilename(filename: string, definitionFile?: boolean): string;
```

Comment:
I don't understand. It is incorrect TypeScript syntax. The default value should be declared in the implementation, not in the type definition. 
If you hover over it to get the information, it does not show the default value as `false` anyway, as `definitionFile: boolean = false`. It shows:

![getUnstyledFilename](https://user-images.githubusercontent.com/20900032/212964505-1c34b719-423c-4b8b-a32e-a6cbb3a3b7eb.PNG)


The CI did not pick it, because `docs-utilities` package is not included during type check step.

Reply:
Won't it error in `.ts` file as well?

Reply:
No, `a: boolean = false` is valid in TypeScript

https://codesandbox.io/s/date-and-time-pickers-forked-l2vp3l?file=/src/test2.d.ts

But as you said, it is not in `.d.ts` files

Reply:
Oh yes. I had to update the CodeSandbox: https://codesandbox.io/s/date-and-time-pickers-forked-l2vp3l?file=/src/test.ts. The `.ts` files must also contain the implementation, which is why it works.

I wonder if I should migrate it to TypeScript.

Reply:
Migrated to TypeScript.